rb/spec/integration/selenium/webdriver/bidi/log_inspector_spec.rb

```diff
@@ -21,13 +21,13 @@
 
 module Selenium
   module WebDriver
-    class BiDi
-      describe LogInspector, exclusive: {browser: %i[chrome firefox]} do
+    class BiDi # rubocop:disable Metrics/ClassLength
```

Comment:
let's add this file path to the exclude list in .rubocop.yml instead of in the code here.